packages/legacy-credentials/src/Credential.ts

```diff
@@ -21,12 +21,13 @@ import { ConfigService } from '@kiltprotocol/config'
 import { Attestation, CType } from '@kiltprotocol/core'
 import {
   isDidSignature,
+  resolve,
   dereference,
   signatureFromJson,
-  signatureToJson,
   verifyDidSignature,
 } from '@kiltprotocol/did'
 import type {
+  DidUrl,
```

Comment:
Why?

Reply:
I find it pretty confusing to have both a `DidUri` and `DidUrl` type

Reply:
Becuase it IS confusing also in the spec 😃 I think it would defeat the purpose of [this PR](https://github.com/KILTprotocol/sdk-js/pull/799) (on which this is built on top of) if we would start, once again, deviating from the terms used in the spec. That's why the renaming.

Reply:
this will not be solved here, but on Antonio's linked PR